src/Components/Artwork/Details.tsx

```diff
@@ -461,9 +477,13 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
           display
         }
       }
+      consignmentSubmission {
```

Comment:
I am worried this might make all artwork grids performance slower - maybe we can only query for this if we are in my collection. Check out this https://relay.dev/docs/guides/alias-directive/#skip-and-include